.github/workflows/CI.yml

```diff
@@ -0,0 +1,82 @@
+name: CI
+
+# Variables
+env:
+  BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
+
+# Triggering the workflow on PRs and pushes to relevant branches
+on:
+  pull_request:
+    branches:
+      - main
+      - feature/**
```

Comment:
We shouldn't run `bump-version-update-changelog` on PRs only when merged to `main`